### PR TITLE
docs: update readme (install seqrepo, psql cmd, + submodules)

### PR DESCRIPTION
**Pull request by korikuzma**
I haven't tried installing UTA locally, but this is what I noticed when cloning the repo, installing a local seqrepo, + testing out psql command

### CODE REVIEW

**Review by theferrit32 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -138,8 +139,8 @@ nearly instantaneous.
 You can test UTA and seqrepo installations like so:
 
 ```shell
-$ psql -XAt postgres://anonymous@localhost/uta -c 'select count(*) from transcript'
-249909
+$ psql -XAt postgres://anonymous@localhost/uta -c 'select count(*) from uta_20210129b.transcript'
```
Comment (theferrit32) [inline README.md:142]: Does this connection string need to include the password added here?

https://github.com/ga4gh/vrs-python/pull/405/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R27

Comment (korikuzma) [reply README.md:142]: @theferrit32 I was able to run the command without the password included

Comment (korikuzma) [reply README.md:142]: @theferrit32 I _think_ it's because anonymous user doesn't have a password and that's why it's not needed

Comment (theferrit32) [reply README.md:142]: @korikuzma got it, just tried it on my laptop and it also worked

**Pull request merged**